content/en/blog/posts/2024-07-15-vulnerability-scanning.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Building Secure Container images with Shipwright"
+date: 2024-07-15T14:20:01-04:00
```

Comment:
Something to note - the deploy preview won't show the blog post if the post date is in the future.